src/QMCWaveFunctions/TrialWaveFunction.h

```diff
@@ -539,6 +540,9 @@ class TrialWaveFunction
   /// spomap_ reference accessor
   const SPOMap& getSPOMap() const { return *spomap_; }
 
+  /// find SD WFCs if exist
+  RefVector<SlaterDet> findSD() const;
```

Comment:
This is note for myself. Not a critical issue of this PR and can be dealt separately.
I'm thinking of adding findComponentsByCategory(category) a more generic function.

Reply:
Noted.